libmavconn/src/interface.cpp

```diff
@@ -392,6 +392,8 @@ MAVConnInterface::Ptr MAVConnInterface::open_url(std::string url,
 		return url_parse_udp(host, query, system_id, component_id, false);
 	else if (proto == "udp-b")
 		return url_parse_udp(host, query, system_id, component_id, true);
+	else if (proto == "udp-bb")
+		return url_parse_udp(host, query, system_id, component_id, true, true);
```

Comment:
In that internal funcs better to be more explicit. I mean always pass permanent-broadcast.

Reply:
I guess, the only explicit way to pass booleans is Pythonic `foo(bar=True, baz=False)`, sadly this is not Python :)